script/update-content-versions-and-check-for-archived-slugs.ts

```diff
@@ -0,0 +1,112 @@
+import fs from 'fs';
+import path from 'path';
+import { argv } from 'yargs';
+import { content } from '../src/app/content/routes';
+import { LinkedArchiveTreeNode } from '../src/app/content/types';
+import { flattenArchiveTree } from '../src/app/content/utils';
+import { makeUnifiedBookLoader } from '../src/app/content/utils';
+import { findArchiveTreeNodeById } from '../src/app/content/utils/archiveTreeUtils';
+import { ARCHIVE_URL, REACT_APP_ARCHIVE_URL, REACT_APP_OS_WEB_API_URL } from '../src/config';
+import books from '../src/config.books';
+import createArchiveLoader from '../src/gateways/createArchiveLoader';
+import createOSWebLoader from '../src/gateways/createOSWebLoader';
+import { Redirects } from '../src/redirects/types';
+
+const { bookId, newVersion } = argv as any as {
+  bookId: string
+  newVersion: string | number;
+};
+
+const booksPath = path.resolve(__dirname, '../src/config.books.json');
+const redirectsPath = path.resolve(__dirname, '../src/redirects/');
+
+const bookLoader = makeUnifiedBookLoader(
+  createArchiveLoader(`${ARCHIVE_URL}${REACT_APP_ARCHIVE_URL}`),
+  createOSWebLoader(`${ARCHIVE_URL}${REACT_APP_OS_WEB_API_URL}`)
+);
+
+async function updateRedirections(_bookId: string, currentVersion: string, _newVersion: string) {
+  if (currentVersion === _newVersion) {
```

Comment:
all these variables with the `_` on the front, the `_` is used to suppress the unused variable warning, if you're using them you shouldn't start with the underscore